apps/crn-frontend/src/shared-research/ResearchOutput.tsx

```diff
@@ -34,9 +44,13 @@ const ResearchOutput: React.FC = () => {
   if (researchOutputData) {
     return (
       <ResearchOutputPermissionsContext.Provider value={permissions}>
-        <Route exact path={path}>
+        <Route exact path={publishedNow ? publishedNowPath : path}>
```

Comment:
Can't you just use an array on path?

Reply:
Issue is that publishedNow is seen as undefined.